sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations/VirtualMachineScaleSetVMsOperations.cs

```diff
@@ -23,42 +14,69 @@ namespace Microsoft.Azure.Management.Compute
     using System.Threading.Tasks;
 
     /// <summary>
-    /// CloudServiceRolesOperations operations.
+    /// VirtualMachineScaleSetVMsOperations operations.
```

Comment:
When you add overloads to an operations class through customizations, you also need to add analogous overloads to the matching methods in the underlying interface and extensions (IVirtualMachineScaleSetVmsOperations, VirtualMachinesScaleSetVMsOperations).  You can do this analogously, and as I've indicated in a few places below, you can simply call the generated code version of the method (with new parameters set to null), rather than duplicating the method body.

(1)

Reply:
@markcowl , added customizations for VMSSVMsOperationsExtensions here: https://github.com/Azure/azure-sdk-for-net/blob/compute-2020-12/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations/VirtualMachineScaleSetVMsOperationsExtensions.cs